motor-control-module/src/main/java/org/sagebionetworks/research/motor_control_module/widget/TapButton.java

```diff
@@ -35,7 +35,8 @@
 import android.content.Context;
 import android.support.v7.widget.AppCompatButton;
 import android.util.AttributeSet;
-import org.sagebionetworks.research.mobile_ui.R;
+
```

Comment:
This was in `drawables-v24` in the `mobile-ui` module!

Reply:
Why in the world is there even a custom class that just sets a drawable?  This should be removed completely